src/plugins/unified_histogram/public/layout/hooks/use_lens_suggestions.ts

```diff
@@ -36,29 +36,28 @@ export const useLensSuggestions = ({
       contextualFields: columns,
       query: query && isOfAggregateQueryType(query) ? query : undefined,
     };
-    const lensSuggestions = isPlainRecord ? lensSuggestionsApi(context, dataView) : undefined;
-    const firstSuggestion = lensSuggestions?.length ? lensSuggestions[0] : undefined;
-    const restSuggestions = lensSuggestions?.filter((sug) => {
-      return !sug.hide && sug.visualizationId !== 'lnsLegacyMetric';
-    });
-    const firstSuggestionExists = restSuggestions?.find(
+    const allSuggestions = isPlainRecord
+      ? lensSuggestionsApi(context, dataView, ['lnsDatatable'])
+      : undefined;
+    const firstSuggestion = allSuggestions?.length ? allSuggestions[0] : undefined;
+    const firstSuggestionExists = allSuggestions?.find(
```

Comment:
Why this check? `firstSuggestion` was the first entry in `allSuggestions`, so this check looks superflous to me as just checking the existence of `firstSuggestion` should suffice

Reply:
It made sense before but not now, I will drop it.

Reply:
Fixed https://github.com/elastic/kibana/pull/153931/commits/fd12a4654c7624cafd1e955555f06334ad55df81